lib/game_controller.rb

```diff
@@ -2,30 +2,43 @@
 require_relative 'turn'
 require_relative 'player'
 require_relative 'computer'
+require_relative 'player_two'
 
 class GameController
-  def start_game
-    board = Board.new
-    player = Player.new
-    computer = Computer.new
+  def choose_game_type
     puts "Welcome to Connect 4!!!"
     puts "Enter P to start a game, or Q to quit."
     user_input = gets.chomp.downcase
-    while user_input != "p" && "q"
+    while user_input != "p" && user_input != "q"
```

Comment:
Was this causing issues or just meant to be more readable?

Reply:
It was giving me "string literal" warning. Did a little research and also remember running into this issue with other code in the past and it's better to explicitly state what the variable does or doesn't equal two with ||  / &&. All in all it was just producing a warning, and not an error, which was something new.